docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -433,8 +434,11 @@ openssl pkcs8 -topk8 -nocrypt -inform pem -in rsaPrivateKey.pem -outform pem -ou
 You can use the generated key pair instead of those used in this quickstart.
 ====
 
-Now, you can generate a JWT to use with the `TokenSecuredResource` endpoint.
-To do this, run the following command:
+To generate a JSON Web Token (JWT) for the `TokenSecuredResource` endpoint, ensure that the application is running.
+
+To do this, make sure to <<run-application>> first.
```

Comment:
Sorry - this became clearer to me today:
```suggestion
Ensure the <<run-application, application is running>> before generating the JSON Web Token (JWT) for the `TokenSecuredResource` endpoint.
```
Or if it seems probable that the application isn't running.
```suggestion
<<run-application>> before generating the JSON Web Token (JWT) for the `TokenSecuredResource` endpoint.
```